edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_buttons.scss

```
@@ -0,0 +1,15 @@
+// General styles for buttons.
+
+// Reset platform styles.
+input[type="submit"], input[type="button"], button, .button {
```

```suggestion
input[type="submit"],
input[type="button"],
button,
.button {
```
Would't it be better to break the lines to separate each individual selector?